commands_Siddhant.txt

```diff
@@ -0,0 +1,47 @@
+1.Create and Checkout a New Branch
+
+#branches from currently checked out directory
+
+git checkout -b <branchName>
+
+2.Configure folder
+
+git config --global user.name "Siddhant"
+
+git config --global user.email siddhant@example.com
+
+
+3.Create a new local repository
```

Comment:
i.e to Configure the author name and email address to be used with commits.

On Wed, Jul 1, 2015 at 11:04 AM, harshal-webonise <notifications@github.com>
wrote:

> In commands_Siddhant.txt
> <https://github.com/Siddhant-webonise/siddhant-assignments/pull/1#discussion_r33649580>
> :
>
> > @@ -0,0 +1,47 @@
> > +1.Create and Checkout a New Branch
> > +
> > +#branches from currently checked out directory
> > +
> > +git checkout -b <branchName>
> > +
> > +2.Configure folder
> > +
> > +git config --global user.name "Siddhant"
> > +
> > +git config --global user.email siddhant@example.com
> > +
> > +
> > +3.Create a new local repository
>
> Add proper description
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Siddhant-webonise/siddhant-assignments/pull/1/files#r33649580>
> .
>